### ISSUE TITLE
Screen Resolutions

### ISSUE THREAD

**Issue by kostaskas** [THEDEVIOUS1/CHUWI-MINIBOOK-HACKINTOSH#3]
Is it possible to add more resolutions on Minibook? THe HiDPI is too big and the normal resolution is so small..
Have tried everything in order to work but no luck

**Comment by THEDEVIOUS1** [comment]
i believe both of those issues u openened are actually caused by the same thing(so will only be replying to one and may close the other) which is the fact that the screen in our device is native portrait instead of landscape as it is being utilized....this makes both adding new resolutions(which does indeed work BUT they only get added to the normal/non rotated list) and fixing clover difficult

**Comment by kostaskas** [comment]
Yes I understood, using Clover Configurator on Hackintosh, I have found an option "vertical" on Clover themes, I checked it but it doesn't seem it's saved, every time I save and rerun Confirurator, "vertical" is unchecked. Wonder if that makes theme to rotate correctly. For it's just the only "problem", if Clover is rotated, you can boot every OS from there (I have already done it Tetraboot, with Windows 10, Ubuntu, Hackintosh and Android x86)

Closed the other issue

**Comment by THEDEVIOUS1** [comment]
Bet i got u beat with my win10, android, linux, hackintosh AND chrome Penta Boot....lol. actually...im running both mojave and catalina so techinically its a sexta(?)boot. but back to business..im looking into ways to alter the edid to trick the system into thinking there are other resolutions available but no luck so far

**Comment by kostaskas** [comment]
I'm just trying to find something to make sd card reader working, I will inform you to add if I find it

**Comment by Azkali** [comment]
Might be the same issue as [this one](https://github.com/Azkali/GPD-P2-MAX-Hackintosh/issues/10), @THEDEVIOUS1 you might need to contact CHUWI directly

**Comment by kostaskas** [comment]
> Bet i got u beat with my win10, android, linux, hackintosh AND chrome Penta Boot....lol. actually...im running both mojave and catalina so techinically its a sexta(?)boot. but back to business..im looking into ways to alter the edid to trick the system into thinking there are other resolutions available but no luck so far

A little of topic, to boot on all systems,  you just choose the OS from Bios or from the bootloader?

**Comment by THEDEVIOUS1** [comment]
actually....im using manajaro's grub to boot everything however i had to manually add macos and android to the list

**Comment by THEDEVIOUS1** [comment]
totally unrelated to this issue...if u want to help test the touchscreen then come join us in [this gitter room](https://gitter.im/lazd/VoodooI2CGoodix)

**Comment by hatran0884** [comment]
@kostaskas have you found a way to add driver for sd card reader ?
and how about the custom resolution ?

**Comment by kostaskas** [comment]
> @kostaskas have you found a way to add driver for sd card reader ?
> and how about the custom resolution ?

I have tried every kext found online,  even for Realtek devices but no luck. Not have much time to check kext. 

Tried to add resolution manually,  but only every timer boot was damaged and macos didn't even start and must reinstall mac
